.ddev/commands/web/clone

```diff
@@ -1,21 +1,52 @@
 #!/bin/bash
 
 ## Description: Checkout a module using git clone
-## Usage: clone MODULE_NAME [-y]
-## Example: "ddev clone admin_toolbar" or "ddev clone admin_toolbar -y"
+## Usage: clone [-b BRANCH_NAME | -t TAG_NAME] [-y] MODULE_NAME
+## Example: "ddev clone admin_toolbar" or "ddev clone -b develop -y admin_toolbar"
```

Comment:
Example text uses `-y` but flag is `-c`

Reply:
Is it possible for the flags to go after the module? I like the idea of having the required element first.

Reply:
Yes, that should be no problem.

Reply:
I updated the comments and help section to reflect that the user should specify the module name first. I also added lines 16-44—these enable the user to place the module name anywhere in the call to `ddev clone` should they not place it first.

It will still error out if they pass the arguments incorrectly.